Update for 2022-04-26

thuiskens opened the pull request.

thuiskens committed:
Update for 2022-03-29

- Add journal club hosted by Leila

thuiskens committed:
Update for 2022-04-12

- Add presentation by Francesca

moorepants commented on whosnext.py:
```diff
@@ -70,6 +70,8 @@
     '2022-03-01': ['Rado Dukalski'],
     '2022-03-15': ['Marco Reijne'],
+    '2022-03-29': ['Leila Alizadehsaravi'],
+    '2022-04-12': ['Francesca Andretta'],
```
Might as well add Jeswin too, so we can run it Tuesday.

thuiskens replied:
I didn't commit his presentation yet in case he would get sick or wouldn't be able to present otherwise, but I added him before running the whosnext today.

thuiskens committed:
Update for 2022-04-26

- Add presentation by Jeswin
- Remove Eline, Koen and Shannon from current lab member list

thuiskens commented:
The latest commit does contain Jeswin's presentation and removes Eline, Koen and Shannon from the current lab member list.
